src/base/bootstrap-overrides/_core-tables.scss

```diff
@@ -12,3 +12,9 @@ caption {
 	}
 	text-align: center;
 }
+
+// Not ported from Bootstrap 4 but inline with Bootstrap 4
+.table-responsive {
+	-webkit-overflow-scrolling: touch;
```

Comment:
webkit prefix is usually not needed. Our PostCSS task usually take care of that. Is there an equivalent formal CSS that do the same?